### PR TITLE
[EYE-105] Fixed non-serializable bug

### PR DESCRIPTION
**Pull request by matthew-latanafrancia**
# Description

Made changes to the notification screen so that when a user is taken to that screen, non-serializable warnings wouldn't occur.  This was done by throwing camera ids to the notification screen since we're always gonna be getting all of a cameras notifications at the very least.  I also included small changes like long month printing on each notification.

## Testing

Everything works as expected and the warnings don't pop up anymore.

## JIRA Ticket Link
https://eyespycamera.atlassian.net/jira/software/projects/EYE/boards/1?selectedIssue=EYE-105

### CODE REVIEW

File: app/human-detector-app/screens/NotifScreen.tsx [hunk 1]
```diff
@@ -31,11 +34,20 @@ export default function NotifScreen({ navigation, route }: Props): React.ReactEl
               }}
             >
               <Text style={styles.dateText}>
-                {notification.timestamp.toLocaleDateString(undefined)}
+                {today.toLocaleDateString(undefined) ===
+                notification.timestamp.toLocaleDateString(undefined)
+                  ? 'Today'
```
Comment (matthew-latanafrancia) [inline app/human-detector-app/screens/NotifScreen.tsx:39]: Is there a better way to check if the date is today?

Comment (1Revenger1) [reply app/human-detector-app/screens/NotifScreen.tsx:39]: Probably just checking year, date, and month. L

Comment (matthew-latanafrancia) [reply app/human-detector-app/screens/NotifScreen.tsx:39]: man

Comment (tucker-moore) [inline app/human-detector-app/screens/NotifScreen.tsx:39]: On the topic of the date debate from Discord:
```suggestion
                {today.getDay() === notification.getDay() ? 'Today'
```

File: app/human-detector-app/__tests__/BackendService.test.ts [hunk 2]
```diff
@@ -126,20 +126,8 @@ describe(BackendService, () => {
...
-      );
-      const notif2: Notification = new Notification(
-        'a',
-        '2007-12-27T00:00:00',
-        'wawa',
-        'groupId',
-        'cameraId'
-      );
+      const notif1: Notification = new Notification('e', '2022-06-14T03:24:00', 'bee', 'cameraId');
+      const notif2: Notification = new Notification('a', '2007-12-27T00:00:00', 'wawa', 'cameraId');
```
Comment (tucker-moore) [inline app/human-detector-app/__tests__/BackendService.test.ts:130]: wawa

**Review by tucker-moore — APPROVED** [review]

File: app/human-detector-app/__tests__/BackendService.test.ts (hunk 2 above)
Comment (1Revenger1) [reply app/human-detector-app/__tests__/BackendService.test.ts:130]: wawa

File: app/human-detector-app/screens/NotifScreen.tsx (hunk 1 above)
Comment (1Revenger1) [reply app/human-detector-app/screens/NotifScreen.tsx:39]: What's the range of the day? Is it day in a month, days since epoch?

**Review by 1Revenger1 — APPROVED** [review]

File: app/human-detector-app/screens/NotifScreen.tsx (hunk 1 above)
Comment (tucker-moore) [reply app/human-detector-app/screens/NotifScreen.tsx]: frick oops lol

**Pull request merged**